#!/usr/bin/env python
PACKAGE = "image_proc_chain"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("val_min", int_t, 0, "A parameter for detection and connection", 50,  1,   200)
gen.add("val_max", int_t, 0, "A parameter for detection and connection", 100, 1,   200)
gen.add("sobel_aperture", int_t, 0, "Size of sobel aperture", 3,  3,   7)

exit(gen.generate(PACKAGE, "image_proc_chain", "CannyEdge"))
